src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js

```diff
@@ -36,9 +36,12 @@ export const BasicSliderInput = (props) => {
     ...otherProps
   } = props;
 
+  const min = typeof minValue === 'number' ? minValue : 0;
+  const max = typeof maxValue === 'number' ? maxValue : 100;
+
   const getValue = (value) => {
-    if (value == null || isNaN(value)) {
-      return 0;
+    if (typeof value !== 'number' || isNaN(value)) {
+      return min ?? 0;
```

Comment:
If it's not possible for `min` to be `null` or `undefined`, maybe we can simply return `min` here?

Reply:
Good idea, it must be a relic of first iterations of the refactoring :laughing: thanks! :pray: